drivers/SmartThings/zigbee-button/src/zigbee-multi-button/supported_values.lua

```diff
@@ -24,7 +24,7 @@ local devices = {
     MATCHING_MATRIX = {
       { mfr = "CentraLite", model = "3450-L" },
       { mfr = "CentraLite", model = "3450-L2" },
-      { mfr = "ROBB smarrt", model = "ROB_200-008-0" }
+      { mfr = "ROBB smarrt", model = "ROB_200-008-0" }    
```

Comment:
```suggestion
      { mfr = "ROBB smarrt", model = "ROB_200-008-0" }
```

Reply:
if you delete and change the whitespace in the code, it will not be saved and it will be shown to request a creat pull request patch.
Please check it.

Reply:
I'm not sure what you mean.